pom.xml

```diff
@@ -25,7 +25,8 @@
     </scm>
 
     <properties>
-        <dep.httpcomponents.version>4.4</dep.httpcomponents.version>
+        <dep.httpcomponents.client.version>4.5.14</dep.httpcomponents.client.version>
+        <dep.httpcomponents.core.version>4.4.16</dep.httpcomponents.core.version>
```

Comment:
why not matching versions?

Reply:
They diverged somehow but `httpcomponents.client:4.5.14` is using `httpcomponents.core:4.4.15`: https://mvnrepository.com/artifact/org.apache.httpcomponents/httpcomponents-client